### DISCUSSION TITLE
ValueError: not enough values to unpack (expected 2, got 1)

### DISCUSSION THREAD

**Discussion by sd50pcs2000** [WASasquatch/was-node-suite-comfyui#449] [Q&A]
!!! Exception during processing!!! not enough values to unpack (expected 2, got 1)
Traceback (most recent call last):
  File "D:\ComfyUI\execution.py", line 152, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\ComfyUI\execution.py", line 82, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\ComfyUI\execution.py", line 75, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\ComfyUI\custom_nodes\was-node-suite-comfyui\WAS_Node_Suite.py", line 3262, in image_paste_face
    result_image, result_mask = self.paste_image(tensor2pil(image), tensor2pil(crop_image), crop_data, crop_blending, crop_sharpening)
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\ComfyUI\custom_nodes\was-node-suite-comfyui\WAS_Node_Suite.py", line 3299, in paste_image
    crop_size, (top, left, right, bottom) = crop_data
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: not enough values to unpack (expected 2, got 1)
help me

**Comment by WASasquatch** [comment]
Sounds like your crop data isn't as expected. Are you using batched images? This doesn't work on batch images. There are alternatives that can do batches in [SaltAI custom node](https://github.com/get-salt-AI/SaltAI).

**Comment by sd50pcs2000** [comment]
![屏幕截图 2024-07-29 063736](https://github.com/user-attachments/assets/2aa1f599-5c0b-40c1-aeec-ded6140fbaff)

**Reply by WASasquatch** [reply]
That's not my node, the auto-crop. It likely has different data structure for its cropping.
